### DISCUSSION TITLE
TextMeshPro 컴포넌트를 ts에 가져와서 쓸수 없나요?

### DISCUSSION THREAD

**Discussion by pda017** [naverz/zepeto-studio-kor#562] [5. Q&A]
메뉴얼에 Textmeshpro 패키지를 쓸수 있다고 나와있는데 import에 TMPro 가 존재하지가 않네요.
GetComponent로 가져와서 쓰고 싶은데 방법을 모르겠습니다.

**Comment by andwhy81** [comment]
https://cafe.naver.com/zepetostudio/9

저는 이방법으로 사용하고 있습니다.
